packages/hoppscotch-app/helpers/actions.ts

```diff
@@ -22,6 +22,7 @@ export type HoppAction =
   | "modals.search.toggle" // Shows the search modal
   | "modals.support.toggle" // Shows the support modal
   | "modals.share.toggle" // Shows the share modal
+  | "modals.developerOptions.toggle" // Shows the developer options modal
```

Comment:
Eventhough we have defined this action and a action handler, we don't actively have any bindings to it, so lets just remove this action.